app/lib/tests/resources.server.test.ts

```diff
@@ -0,0 +1,70 @@
+import { replaceRelativeLinks } from "../resources.server";
+
+describe("replaceRelativeLinks", () => {
+  it("should replace relative links with absolute links for images", () => {
+    const input = `<img src="./relative">`;
+    const repoUrl = "https://my-repo";
+    const expected = `<img src="${repoUrl}/raw/main/relative">`;
+    expect(replaceRelativeLinks(input, repoUrl)).toEqual(expected);
+  });
+
+  it("should not replace relative links with absolute links for links", () => {
+    const input = `<link src="./relative">`;
+    const repoUrl = "https://my-repo";
+
+    expect(replaceRelativeLinks(input, repoUrl)).toEqual(input);
+  });
+
+  it("should replace relative links with absolute links for multiple images", () => {
+    const input = `<img src="./relative1"><img src="./relative2">`;
+    const repoUrl = "https://my-repo";
+    const expected = `<img src="${repoUrl}/raw/main/relative1"><img src="${repoUrl}/raw/main/relative2">`;
+    expect(replaceRelativeLinks(input, repoUrl)).toEqual(expected);
+  });
+
+  it("should replace relative links with absolute links for images with other attributes", () => {
+    const input = `<img src="./relative" alt="alt text">`;
+    const repoUrl = "https://my-repo";
+    const expected = `<img src="${repoUrl}/raw/main/relative" alt="alt text">`;
```

Comment:
Unfortunately this breaks if `alt` (or any attribute) comes before `img`

```js
    const input = `<img alt="alt text" src="./relative">`;
    const repoUrl = "https://my-repo";
    const expected = `<img alt="alt text" src="${repoUrl}/raw/main/relative">`;
```

Reply:
good catch, added a failing test, then changed it to work!

Reply:
Fun fact, having an integrated GitHub copilot for this was _super_ nice

<img width="1800" alt="image" src="https://github.com/remix-run/remix-website/assets/12396812/cbeb2125-9adb-49d3-b2e5-1fd1b7ed01af">